src/model.ts

```diff
@@ -233,6 +234,15 @@ export default class Model {
     }
   }
 
+  isMarkedForDisassociation(val? : boolean) : boolean {
```

Comment:
Thoughts on switching this to use getters and setters instead?  Would be a better signal of intent.

Reply:
Not against it, but for this particular PR just want to match the existing api